ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -270,15 +272,44 @@ def continuous(self) -> np.ndarray:
     def discrete(self) -> np.ndarray:
         return self._discrete
 
-    @staticmethod
-    def create_continuous(continuous: np.ndarray) -> "ActionTuple":
-        discrete = np.zeros((continuous.shape[0], 0), dtype=np.int32)
-        return ActionTuple(continuous, discrete)
+    def add_continuous(self, continuous: np.ndarray) -> None:
```

Comment:
Technically, this is not adding, but setting the continuous value. Should this be `set_` or `add_`?